src/tools/flash.c

```diff
@@ -84,18 +84,18 @@ static int get_opts(struct opts* o, int ac, char** av)
     {
         ac--;
         av++;
-	    int i=strlen(av[0]);
-	    if(i%2 != 0){
+	    int j = (int) strlen(av[0]);
```

Comment:
this is not really portable, as strlen really returns `size_t` we need to obey and not cast it to a signed type.